### DISCUSSION TITLE
Webd: A 90KB Alternative to HFS You Might Want to Try

### DISCUSSION THREAD

**Discussion by webd90kb** [rejetto/hfs#679] [Show and tell]
I developed this software years ago, inspired by HFS. 
At that time, I wanted a similar software that could run on Linux/OpenWRT and Android TV boxes.
see https://github.com/webd90kb/webd

**Comment by rejetto** [comment]
why do you keep releases inside git, instead of using "releases" on github?

**Reply by webd90kb** [reply]
Because Webd's binary files are very small, managing them directly through git commands is straightforward, automated, and efficient. While 'releases' on GitHub could potentially be automated as well, I am not familiar with that process.
